os/fs/procfs/fs_procfslogsave.c

```diff
@@ -120,9 +120,12 @@ const struct procfs_operations logsave_operations = {
 static int logsave_open(FAR struct file *filep, FAR const char *relpath, int oflags, mode_t mode)
 {
 	FAR struct logsave_file_s *attr = (struct logsave_file_s *)kmm_zalloc(sizeof(struct logsave_file_s));
-	fvdbg("Open '%s'\n", relpath);
 	filep->f_priv = (FAR void *)attr;
-	log_dump_read_init();
+	if(log_dump_read_wake() != OK) {
+		fvdbg("read init fail\n");
+		return ERROR;
```

Comment:
Is it okay not to close the mq?

Reply:
mq is always kept open for send and receive even if some calls fail and retry